kubernetes/resource_kubernetes_service_account.go

```diff
@@ -258,3 +269,80 @@ func resourceKubernetesServiceAccountExists(d *schema.ResourceData, meta interfa
 	}
 	return true, err
 }
+
+func resourceKubernetesServiceAccountImportState(d *schema.ResourceData, meta interface{}) ([]*schema.ResourceData, error) {
+	conn := meta.(*kubernetes.Clientset)
+
+	namespace, name, err := idParts(d.Id())
+	if err != nil {
+		return nil, err
```

Comment:
Would you mind wrapping these errors before returning them? The idea being to make it obvious that the error happened during import processing, since these can also occur during a normal Read.